packages/repository/src/__tests__/unit/repositories/relations-helpers/relations-helpers-fixtures.ts

```diff
@@ -100,3 +161,7 @@ export function createCategory(properties: Partial<Category>) {
 export function createProduct(properties: Partial<Product>) {
   return new Product(properties as Product);
 }
+
+export function createManufacturer(properties: Partial<Manufacturer>) {
+  return new Manufacturer(properties as Manufacturer);
```

Comment:
The constructor of `Manufacturer` class takes in a partial manufacturer on [line](https://github.com/strongloop/loopback-next/pull/3721/files#diff-9d83c357b3532fd977f40495748d9a7bR30), any reason why specify the type `as Manufacturer` again?

Reply:
Before we were passing in a single value. Will remove